speedcord/shard.pyi

```diff
@@ -0,0 +1,58 @@
+from speedcord import Client
+from asyncio import AbstractEventLoop, Lock, Event
+from logging import Logger
+
+from aiohttp import ClientWebSocketResponse
+
+
+class DefaultShard:
+    id: int
+    client: Client
+    loop: AbstractEventLoop
+    ws: ClientWebSocketResponse
```

Comment:
`ws` should be optional and removed from shard.py

Reply:
Thought so, but didn't want to make any changes to the main `.py` files, so took them as the main source of truth and followed it's pattern for these `shard.pyi` typings.